packages/features/users/components/UserV2OptInBanner.tsx

```diff
@@ -1,10 +1,16 @@
+import { useEffect, useState } from "react";
+
 function UserV2OptInBanner() {
+  const [v2OptInCookie, setV2OptInCookie] = useState<boolean | null>(null);
   // Only show on client-side
-  if (typeof document === "undefined") return null;
-
-  const hasV2OptInCookie = document.cookie.includes("calcom-v2-early-access=1");
+  useEffect(() => {
+    setV2OptInCookie(document.cookie.includes("calcom-v2-early-access=1"));
+  }, []);
 
-  if (hasV2OptInCookie)
+  if (!v2OptInCookie) {
+    return null;
+  }
+  if (v2OptInCookie)
```

Comment:
Is there a reason for this change? If there's not let's revert it since it's going to be deprecated anyways.